spdb/spatialdb/object.py

```diff
@@ -738,7 +746,9 @@ def trigger_page_out(self, config_data, write_cuboid_key, resource):
         msg_data = {"config": config_data,
                     "write_cuboid_key": write_cuboid_key,
                     "lambda-name": "s3_flush",
-                    "resource": resource.to_dict()}
+                    "resource": resource.to_dict(),
+                    "to_black": str(to_black).lower()
```

Comment:
You should be able to just do `"to_black": to_black`. Then when you call `json.dumps()`, this will be serialized as a boolean value instead of the string "true" or "false".